instance/models/openedx_appserver.py

```diff
@@ -240,8 +243,17 @@ def make_active(self, active=True):
         Parameters:
         * `active`: defaults to True.  Set to False to deactivate the appserver.
         """
-        self.logger.info('Making %s %s for instance %s...',
-                         self.name, "active" if active else "inactive", self.instance.name)
+        self.logger.info(
+            'Making %s %s for instance %s...',
+            self.name,
+            "active" if active else "inactive",
+            self.instance.name
+        )
+        # Try to start/stop services on VM
+        # If this fails, don't block activation, but log output
+        if not self.manage_instance_services(active=active):
```

Comment:
I've changed this to avoid activating an appserver if this fails.

Reply:
Not sure I follow, it will continue and activate the instance, no?

Reply:
I've updated the code to prevent activating appservers when the playbook to activate services fail. This is to avoid adding a broken server to the load balancer pool.